extensions/sql-database-projects/src/models/publishProfile/publishProfile.ts

```diff
@@ -55,6 +56,13 @@ export async function load(profileUri: vscode.Uri, dacfxService: utils.IDacFxSer
 	// get all SQLCMD variables to include from the profile
 	const sqlCmdVariables = utils.readSqlCmdVariables(profileXmlDoc, true);
 
+	TelemetryReporter.createActionEvent(TelemetryViews.SqlProjectPublishDialog, TelemetryActions.profileLoaded)
+		.withAdditionalProperties({
+			hasTargetDbName: (targetDbName !== '').toString(),
```

Comment:
For these is there any possibility of targetDbName/connectionInfo.connectionId being undefined? Probably safer to do falsy checks here just in case.